psh/test.yaml

```
@@ -1,4 +1,7 @@
 test:
+    targets:
+        value: [sparcv8leon3-generic-qemu]
```

You should include target, not set sparcv8leon3-generic-qemu as only target, also comment should be place why it is done.
```suggestion
        include: [sparcv8leon3-generic-qemu]
```